commands/inventory/add_agent_rds_exporter.go

```diff
@@ -24,14 +24,16 @@ import (
 
 var addAgentRDSExporterResultT = commands.ParseTemplate(`
 RDS Exporter added.
-Agent ID              : {{ .Agent.AgentID }}
-PMM-Agent ID          : {{ .Agent.PMMAgentID }}
-Service ID            : {{ .Agent.ServiceID }}
-Listen port           : {{ .Agent.ListenPort }}
+Agent ID                  : {{ .Agent.AgentID }}
+PMM-Agent ID              : {{ .Agent.PMMAgentID }}
+Service ID                : {{ .Agent.ServiceID }}
+Listen port               : {{ .Agent.ListenPort }}
 
-Status                : {{ .Agent.Status }}
-Disabled              : {{ .Agent.Disabled }}
-Custom labels         : {{ .Agent.CustomLabels }}
+Status                    : {{ .Agent.Status }}
+Disabled                  : {{ .Agent.Disabled }}
+Disabled Basic Metrics    : {{ .Agent.DisableBasicMetrics }}
```

Comment:
looks like it should be `.Agent.BasicMetricsDisabled` and `EnhancedMetricsDisabled` here.